src/calculateRentalCost.js

```diff
@@ -5,6 +5,16 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const sum = 40 * days;
+
+  if (sum < 120) {
+    return sum;
+  } else if (sum >= 280) {
+    return sum - 50;
+  } else if (sum >= 120) {
+    return sum - 20;
```

Comment:
This condition is not clear because it's based on the total sum. It should instead be based on the number of days the car is rented.